customer_satisfaction/model/data_ingestion.py

```diff
@@ -0,0 +1,48 @@
+import os
+
+import numpy as np
+import pandas as pd
+
+class IngestData: 
+    '''
+    Data Ingestion class which ingests the data from the source and returns the dataframe. 
+    '''
+    def __init__(self) -> None:
+        pass
+
+    def read_data(self) -> pd.DataFrame:
+        data = pd.read_csv("./data/olist_customers_dataset.csv")
+        order_itemdata = pd.read_csv("./data/olist_order_items_dataset.csv")
+        pay_data = pd.read_csv("./data/olist_order_payments_dataset.csv")
+        rev_data = pd.read_csv("./data/olist_order_reviews_dataset.csv")
+        orders = pd.read_csv("./data/olist_orders_dataset.csv")
+        order_prddata = pd.read_csv("./data/olist_products_dataset.csv")
+        order_prd_catdata = pd.read_csv(
+            "./data/product_category_name_translation.csv"
+        )

```

Comment:
See, These data are taken from kaggle and then combined into single df and finalized as final df, if anyone wants to download full data they can do it but it's not really required, we used this to create final df

Reply:
I think kaggle allows you to download these datasets without login using wget, no? If so it would still be nice to give the user the full experience